### PR TITLE
Add a README

### PR DESCRIPTION
**Pull request by sgugger**
This PR adds a README to the repo, while we wait for a more fledged-out documentation.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,5 +1,175 @@
...
+
+```
+    Args:
+        input_ids (`torch.LongTensor` of shape `(batch_size, sequence_length)`):
+            Indices of input sequence tokens in the vocabulary.
+
+            Indices can be obtained using [`AlbertTokenizer`]. See [`~PreTrainedTokenizer.encode`] and
+            [`~PreTrainedTokenizer.__call__`] for details.
+
+            [What are input IDs?](../glossary#input-ids)
```
Comment (mishig25) [inline README.md:131]: here could you also add one more argument whose type is a custom class like to show that it is possible to do so:
```
some_arg ([`~transformers.SomeClass`])
```

**Comment by mishig25** [comment]
You can also provide info about the imgs:
1. Upload your img to https://huggingface.co/datasets/huggingface/documentation-images
2. And in your mdx file, use like this:
```html
<div class="flex justify-center">
    <img src="https://huggingface.co/datasets/huggingface/documentation-images/resolve/main/datasets/upload_files.png"/>
</div>
```
3. Optionally, you can provide `lightmode` & `darkmode` for the img using syntax below
```html
<div class="flex justify-center">
    <img class="block dark:hidden" src="https://huggingface.co/datasets/huggingface/documentation-images/resolve/main/datasets/resample.gif"/>
    <img class="hidden dark:block" src="https://huggingface.co/datasets/huggingface/documentation-images/resolve/main/datasets/resample-dark.gif"/>
</div>
```

**Comment by sgugger** [comment]
I didn't include the part about the images as it's not really available to external contributors (they can't upload to the dataset).

**Review by mishig25 — APPROVED** [review]
lgtm!

1. Agree to exclude the imgs instructions since community members can't do so\
2. I'll let you decide [on this one](https://github.com/huggingface/doc-builder/pull/144#discussion_r827414142)

**Review by LysandreJik — APPROVED** [review]
This looks good! I see no mention of the `<Tip>` keyword, should it be mentioned? Are there other keywords that should be mentioned? I'm thinking of `Raises` and `Yields`. 

I'm also happy to complete the documentation in a next step if you'd like to get this through quickly.

**Comment by sgugger** [comment]
Thanks for the reviews! I addressed your comments in the last commit, with the exception of `Raises`/`Yields`. I feel we should only document those once they are working.

**Pull request merged**